lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -4,8 +4,18 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 'use strict';
+/* global window */
+
+/** @type {typeof import('url').URLSearchParams} */
+let URLSearchParams;
+if (typeof window !== 'undefined') {
```

Comment:
does this need to be done now? is it already being run in a browser at this point?

Reply:
Yea, I snuck this in here :) Could remove if you like.

No, nothing running in a browser yet.

Reply:
FWIW, `URLSearchParams` is a [global starting in Node 10](https://nodejs.org/api/url.html#url_class_urlsearchparams), so neither environment needs this declaration anymore, they can just use it

Reply:
cool beans let's remove then :)

Reply:
neat